File: bitget-java-sdk-api/src/main/java/com/bitget/openapi/api/v2/MixMarketApi.java
package com.bitget.openapi.api.v2;

import com.bitget.openapi.dto.response.ResponseResult;
import retrofit2.Call;
import retrofit2.http.GET;
import retrofit2.http.QueryMap;

import java.util.Map;

public interface MixMarketApi {

    @GET("/api/v2/mix/market/contracts")
    Call<ResponseResult> contracts(@QueryMap Map<String, String> paramMap);

    @GET("/api/v2/mix/market/orderbook")
    Call<ResponseResult> orderbook(@QueryMap Map<String, String> paramMap);

    @GET("/api/v2/mix/market/ticker")
    Call<ResponseResult> ticker(@QueryMap Map<String, String> paramMap);

    @GET("/api/v2/mix/market/tickers")
    Call<ResponseResult> tickers(@QueryMap Map<String, String> paramMap);

    @GET("/api/v2/mix/market/fills")
    Call<ResponseResult> fills(@QueryMap Map<String, String> paramMap);

    @GET("/api/v2/mix/market/candles")
    Call<ResponseResult> candles(@QueryMap Map<String, String> paramMap);

}

File: bitget-php-sdk-api/src/api/v2/MixMarketApi.php
<?php


namespace bitget\api\v2;


use bitget\internal\BitgetApiClient;

class MixMarketApi
{
    var BitgetApiClient $BitgetApiClient;

    public function __construct($BitgetApiClient)
    {
        $this->BitgetApiClient = $BitgetApiClient;
    }

    public function contracts($params): string
    {
        return $this->BitgetApiClient->doGet("/api/v2/mix/market/contracts", $params);
    }

    public function orderbook($params): string
    {
        return $this->BitgetApiClient->doGet("/api/v2/mix/market/orderbook", $params);
    }

    public function ticker($params): string
    {
        return $this->BitgetApiClient->doGet("/api/v2/mix/market/ticker", $params);
    }

    public function tickers($params): string
    {
        return $this->BitgetApiClient->doGet("/api/v2/mix/market/tickers", $params);
    }

    public function fills($params): string
    {
        return $this->BitgetApiClient->doGet("/api/v2/mix/market/fills", $params);
    }

    public function candles($params): string
    {
        return $this->BitgetApiClient->doGet("/api/v2/mix/market/candles", $params);
    }
}